lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/components/GraalVMComponentConfiguration.scala

```diff
@@ -14,10 +14,13 @@ class GraalVMComponentConfiguration extends RuntimeComponentConfiguration {
   ): Seq[GraalVMComponent] = {
     val optPythonComponent =
       if (os.hasPythonSupport) Seq(GraalVMComponent.python) else Seq()
+    val optRComponent =
+      if (os.hasRSupport) Seq(GraalVMComponent.R) else Seq()
```

Comment:
This is a separate fix that I noticed recently from @xvcgreg's logs from another issue. Apparently, we try to install FastR on Windows, but that is not possible - it is not supported on Windows. Same case as with GraalPy, but FastR will, most probably, never be supported on Windows.